packages/@vue/cli-plugin-unit-jest/generator/index.js

```diff
@@ -79,6 +79,12 @@ const applyTS = module.exports.applyTS = (api, invoking) => {
       moduleFileExtensions: ['ts', 'tsx'],
       transform: {
         '^.+\\.tsx?$': 'ts-jest'
+      },
+      global: !api.hasPlugin('babel') ? undefined : {
+        'ts-jest': {
+          // we need babel to transpile JSX
+          babelConfig: true
```

Comment:
According to `ts-jest` this option will

>  ts-jest will try to find a .babelrc, .babelrc.js file or a babel section in the package.json ...

But vue-cli generates a `babel.config.js`, a relatively new way to configure babel.

`ts-jest` lets you specify the path like so:

```jsglobals: {
  'ts-jest': {
    babelConfig: 'babel.config.js'
  }
}
```

Maybe that would work.

Reply:
Thanks, but `true` actually worked like a charm in a vue-cli project I tried it with. So I don'T think we need to specifiy a path./filename

Reply:
huh, awesome. I wonder why that worked. Good to hear!